src/test/java/net/snowflake/ingest/connection/SecurityManagerTest.java

```diff
@@ -41,19 +47,45 @@ public void testSecurityManagerInstantiate() {
         return;
       }
     }
-    assertFalse("testSecurityManagerInstantiate failed", true);
+    assertFalse("testJWTManagerInstantiate failed", true);
+  }
+
+  /** Test instantiate OAuth manager with invalid params */
+  @Test
```

Comment:
You can use
@Test(expected = IllegalArgumentException.class) 
for test cases throws the correct exception